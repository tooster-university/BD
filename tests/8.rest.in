{"open": {"database": "student", "login": "app", "password": "qwerty"}}
{"support": {"timestamp": 66183751, "member": 8, "password": "1B5FNJ5", "action": 134, "project": 335, "authority": 383}}
{"support": {"timestamp": 125635460, "member": 6, "password": "HMEQ1", "action": 174, "project": 344, "authority": 428}}
{"trolls": {"timestamp": 140938646}}
{"protest": {"timestamp": 151706110, "member": 6, "password": "HMEQ1", "action": 135, "project": 303, "authority": 454}}
{"support": {"timestamp": 153372478, "member": 2, "password": "87PZYU9", "action": 117, "project": 303, "authority": 445}}
{"votes": {"timestamp": 178395804, "member": 2, "password": "87PZYU9"}}
{"downvote": {"timestamp": 183460527, "member": 2, "password": "87PZYU9", "action": 192}}
{"support": {"timestamp": 212040532, "member": 6, "password": "HMEQ1", "action": 198, "project": 296, "authority": 381}}
{"upvote": {"timestamp": 263663507, "member": 2, "password": "87PZYU9", "action": 195}}
{"projects": {"timestamp": 266300244, "member": 7, "password": "EAE5BR3HLB"}}
{"votes": {"timestamp": 312235779, "member": 6, "password": "HMEQ1"}}
{"actions": {"timestamp": 332988676, "member": 7, "password": "EAE5BR3HLB"}}
{"trolls": {"timestamp": 359676974}}
{"votes": {"timestamp": 392425175, "member": 8, "password": "1B5FNJ5"}}
