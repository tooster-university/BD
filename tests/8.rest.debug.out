{"status": "OK"}
{"status": "OK"}
{"status": "OK"}
{"status": "OK", "data": []}
{"status": "OK"}
{"status": "ERROR", "debug": "ERROR:  Member inactive"}
{"status": "ERROR", "debug": "ERROR:  Member inactive"}
{"status": "ERROR", "debug": "ERROR:  Member inactive"}
{"status": "ERROR", "debug": "ERROR:  Member inactive"}
{"status": "ERROR", "debug": "ERROR:  Member inactive"}
{"status": "ERROR", "debug": "ERROR:  Member must be leader but doesnt exist"}
{"status": "ERROR", "debug": "ERROR:  Member inactive"}
{"status": "ERROR", "debug": "ERROR:  Member must be leader but doesnt exist"}
{"status": "OK", "data": []}
{"status": "ERROR", "debug": "ERROR:  Member inactive"}
